src/hoverifier.ts

```diff
@@ -335,12 +335,16 @@ export const TOOLTIP_DISPLAY_DELAY = 100
 export const MOUSEOVER_DELAY = 50
 
 /**
+ * Function that returns a Subscribable of the hover result to be shown.
+ * It may emit on the Observable to update the content and must indicate when it starts and stopped loading new content.
```

Comment:
```suggestion
 * If a Subscribable is returned, it may emit more than once to update the content,
 * and must indicate when it starts and stopped loading new content.
```